.github/workflows/publish-major-minor.yml

```
@@ -0,0 +1,33 @@
+name: "Publish Major-Minor-Tags"
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  push-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install packaging
+        run: pip install -U packaging
```

I prefer being a bit more precise here:
```suggestion
        run: python -m pip install -U packaging --user
```